doc/api/perf_hooks.md

```diff
@@ -521,6 +521,24 @@ property will be an {Object} with two properties:
   * `perf_hooks.constants.NODE_PERFORMANCE_GC_FLAGS_ALL_EXTERNAL_MEMORY`
   * `perf_hooks.constants.NODE_PERFORMANCE_GC_FLAGS_SCHEDULE_IDLE`
 
+### HTTP ('http') Details
+
+This could add additional memory overhead and should only be used for
```

Comment:
I would move this at the end of the section.

Reply:
> I would move this at the end of the section.

Done.